rasa_core/events/__init__.py

```diff
@@ -560,6 +560,46 @@ def _from_story_string(
                                   parameters.get("timestamp"))]
 
 
+# noinspection PyProtectedMember
+class ReminderCancelled(Event):
+    """ Cancel all jobs with a specific name."""
+
+    type_name = "cancel_reminder"
+
+    def __init__(self, action_name, name=None, timestamp=None):
```

Comment:
Based on your comment of the danger of removing for all the users, let's remove `name` here, sorry for inconvenience
```suggestion
    def __init__(self, action_name, timestamp=None):
```

Reply:
Changed it already in my last commit